public/app/percona/backup/components/AddBackupPage/AddBackupPage.tsx

```diff
@@ -271,13 +271,15 @@ const AddBackupPage: FC<GrafanaRouteComponentProps<{ type: string; id: string }>
                           )}
                         </Field>
                       </span>
-                      <span className={styles.wideField}>
-                        <TextareaInputField
-                          fieldClassName={styles.textAreaField}
-                          name="description"
-                          label={Messages.description}
-                        />
-                      </span>
+                      {scheduleMode && (
```

Comment:
only scheduled mode accepts description?

Reply:
Yeah, exactly. On the backend, they are saving it only for scheduled backups.